src/middlewares/checkArticlePemissions.js

```diff
@@ -0,0 +1,16 @@
+import status from '../config/status';
+
+export const canDeleteOrUnpublishArticle = (req, res) => (req.user.id === req.article.userId || req.user.role === 'admin'
```

Comment:
In this middleware, I see that errors messages are the same, to avoid duplication I think it's good to combine them

Reply:
@rwajon check, it's updated